Jenkinsfile

```diff
@@ -141,6 +141,7 @@ node ('master') {
   testing = true
   testList = '(module)'
   fuzzing = true // testing = true
+  benchmarking = true // testing = true
```

Comment:
```suggestion
  benchmarking = testing
```

Reply:
Technically benchmarking could be off while testing is on. I guess it's unnecessary to make benchmarking always on if testing is on.
I decided to leave a commentary here to notice that benchmarking will be built only if testing is on.